tests/PolicyTest.php

```diff
@@ -65,6 +65,12 @@ public function testAReportURICanBeSet(): void
         $policy->applyTo($response);
         $this->assertContains('report-to https://silverstripe.com', $response->getHeader('Content-Security-Policy-Report-Only'));
         $this->assertContains('report-uri https://silverstripe.com', $response->getHeader('Content-Security-Policy-Report-Only'));
+
+        $policy->reportTo('https://silverstripe.com', 'https://silverstripe.com/uri');
```

Comment:
Perhaps you could add this as a separate test suite to check the scenario where the `report-uri` is not set to verify that it will use the `report-to` value?